### PR TITLE
Add a first snap flow for pre-built apps

### PR DESCRIPTION
**Pull request by evandandrea**
Add a first snap flow for pre-built apps.

@sergiusens @popey @flexiondotorg @igorljubuncic @degville I'd appreciate any copy review.

### CODE REVIEW

**Comment by Lukewh** [comment]
You also need to change the "Continue" link at `_fsf_pre-built.html:44` to start the FSF

File: webapp/first_snap/content/pre-built/package.yaml [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+download: git clone https://github.com/snapcraft-docs/geekbench4
+createDir: |
+  cd geekbench4
+  $EDITOR snapcraft.yaml
+metadata: |
+  name: geekbench4
+  version: 4.2.0
+  summary: Cross-Platform Benchmark
+  description: |
+    Geekbench 4 measures your system's power and tells you whether your
```
Comment (Lukewh) [inline webapp/first_snap/content/pre-built/package.yaml]: The line breaks here need adjusting as it causes bad wrapping on the site:
screenshot_2018-12-04 snapcraft - snaps are universal linux packages

**Comment by popey** [comment]
The "How geekbench4 defines snapcraft.yaml" text feels weird. I know it's grammatically accurate, but it just looks odd on the page, and I can't quite put my finger on why.

File: webapp/first_snap/content/pre-built/package.yaml (hunk 1 above)
Comment (evandandrea) [reply webapp/first_snap/content/pre-built/package.yaml]: Fixed as 92a4c30.

**Pull request merged**